### PR TITLE
Expand testing matrix to include K3s and Kubernetes [SAME VERSION]

### PR DESCRIPTION
**Pull request by bfjelds**

### CODE REVIEW

File: .github/workflows/run-test-cases.yml [hunk 1]
```diff
@@ -44,39 +48,49 @@ jobs:
...
+      run: |
+        sudo curl -sfL https://get.k3s.io | sh -
+        sudo addgroup k3s-admin
+        sudo adduser $USER k3s-admin
+        sudo usermod -a -G k3s-admin $USER
+        sudo chgrp k3s-admin /etc/rancher/k3s/k3s.yaml
+        sudo chmod g+r /etc/rancher/k3s/k3s.yaml
+        export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+        echo '--set agent.host.crictl=/usr/local/bin/crictl --set agent.host.dockerShimSock=/run/k3s/containerd/containerd.sock' > /tmp/cri_args_to_test.txt
+        echo 'microk8s kubectl' > /tmp/runtime_cmd_to_test.txt
```
Comment (jiria) [inline .github/workflows/run-test-cases.yml:63]: This line does not look right (microk8s for k3s).

File: .github/workflows/run-test-cases.yml [hunk 2]
```diff
@@ -44,39 +48,49 @@ jobs:
...
+        echo '/etc/rancher/k3s/k3s.yaml' > /tmp/kubeconfig_path_to_test.txt
+
+    - if: ${{ matrix.runtime }} == 'microk8s'
+      continue-on-error: true
+      name: Install K3s
```
Comment (jiria) [inline .github/workflows/run-test-cases.yml:68]: And this should probably say MicroK8s

**Review by jiria — APPROVED** [review]

File: .github/workflows/run-test-cases.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Run End-to-End Scenario
+name: Validate Akri with K3s, Kubernetes, and MicroK8s
```
Comment (kate-goldenring) [inline .github/workflows/run-test-cases.yml:1]: Should we also change where the "Run End-to-End Test" badge on the base readme points?

Comment (bfjelds) [reply .github/workflows/run-test-cases.yml:1]: was always the plan ... just had to know if/how github encoded the commas.

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**